src/main/java/org/kantega/notsoserial/AbstractClassResolverClassVisitor.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2015 Kantega AS
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kantega.notsoserial;
+
+import org.objectweb.asm.ClassVisitor;
+import org.objectweb.asm.MethodVisitor;
+import org.objectweb.asm.Opcodes;
+import org.objectweb.asm.Type;
+
+/**
+ * Add the check for white/black listed classes to JBoss Marshalling (http://jbossmarshalling.jboss.org/). 
+ * 
+ * This class supports version 1.2 up to 1.4.10 of Jboss Marshalling and can be used for both Jboss AS and Wildfly.
+ */
+public class AbstractClassResolverClassVisitor extends ClassVisitor {
```

Comment:
Would it be an idea to prefix this class, and the methods with the similar names, with JBoss? To make it even clearer what it does...